WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -3700,6 +3727,9 @@ This part is the link to the website, and forms part of a longer sentence which
 /* Select parent category screen - Screen title */
 "Select Parent Category" = "Select Parent Category";
 
+/* Button to select specific categories applicable for a coupon in the view for adding or editing a coupon. */
+"Select Product Categories" = "Select Product Categories";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dR3901-R3902)